conf/application.conf

```diff
@@ -294,3 +294,8 @@ bootstrap.http.headersAllowlist = [ "Gov-Test-Scenario", "Location", "X-Request-
 internalServiceHostPatterns = [ "localhost" ]
 
 accessibility-statement.service-path = "/income-tax-view-change"
+
+encryption {
+  key = "QmFyMTIzNDVCYXIxMjM0NQ=="
+  isEnabled = false
+}
```

Comment:
I think this should be stored in app-config-base instead
See here, encryption not to be stored in application.conf
https://github.com/hmrc/app-config-base/blob/main/documentation/README.md#applicationconf

Reply:
This key should be changed as well, given that this repo is public